app/helpers/lists_helper.rb

```diff
@@ -9,10 +9,26 @@ def list_item list, active = false
       "aria-controls" => list.id 
     }
     link_to("#list-#{list.id}", options) do
-      list_name = content_tag :div, list.name, class: "me-auto list-name"
+      if is_shared_list?(list, current_user)
+        list_name = content_tag :div, list.name + " (shared)", class: "me-auto list-name"
+      else
+        list_name = content_tag :div, list.name, class: "me-auto list-name"
+      end
       item_count = content_tag :span, list.items.active.count, class: "badge bg-primary rounded-pill item-count"
 
       list_name + item_count
     end
   end
+
+  def is_shared_list?(list, user)
+    if SharedList.find_by(list_id: list.id)
+      true
+    else
+      false
+    end
```

Comment:
so there's an easier way to determine if the list is a shared list or not.  Instead of checking if there's an entry in the SharedLists table, you can check if the user is the owner of the list.  You won't (and we should add validation that you can't) share with yourself, so if they don't match then it's a shared list.
```suggestion
    list.user_id != user.id
```